R/load_config.R

```diff
@@ -0,0 +1,178 @@
+load_config <- function(config) {
+  if (is.character(config)) return(config::get(file = config))
+  config
+}
+
+get_abcd_dir <- function(params) {
+  params[["directories"]][["dir_abcd"]]
+}
+
+get_abcd_filename <- function(params) {
+  params[["file_names"]][["filename_abcd"]]
+}
+
+get_abcd_sheet <- function(params) {
+  params[["file_names"]][["sheet_abcd"]]
+}
+
+get_matched_dir <- function(params) {
+  params[["directories"]][["dir_matched"]]
+}
+
+get_raw_dir <- function(params) {
+  params[["directories"]][["dir_raw"]]
+}
+
+get_scenario_dir <- function(params) {
+  params[["directories"]][["dir_scenario"]]
+}
+
+get_output_dir <- function(params) {
+  params[["directories"]][["dir_output"]]
+}
+
+get_scenario_tms_filename <- function(params) {
+  params[["file_names"]][["filename_scenario_tms"]]
+}
+
+get_scenario_sda_filename <- function(params) {
+  params[["file_names"]][["filename_scenario_sda"]]
+}
+
+get_start_year <- function(params) {
+  params[["project_parameters"]][["start_year"]]
+}
+
+get_time_frame <- function(params) {
+  params[["project_parameters"]][["time_frame"]]
+}
+
+get_remove_inactive_companies <- function(params) {
+  params[["project_parameters"]][["remove_inactive_companies"]]
+}
+
+get_scenario_source <- function(params) {
+  params[["project_parameters"]][["scenario_source"]]
+}
+
+get_scenario_select <- function(params) {
+  params[["project_parameters"]][["scenario_select"]]
+}
+
+get_region_select <- function(params) {
+  params[["project_parameters"]][["region_select"]]
+}
+
+get_aggregate_alignment_metric_by_group <- function(params) {
```

Comment:
this will have to be renamed once we tackle the more generalized grouping across the modules. but fine for now